Sources/SwiftDocC/Infrastructure/Diagnostics/DiagnosticConsumer.swift

```diff
@@ -17,7 +17,11 @@ public protocol DiagnosticConsumer: AnyObject {
     func receive(_ problems: [Problem])
     
     /// Inform the consumer that the engine has sent all diagnostics for this build.
+    @available(*, deprecated, message: "Please use flush() instead.")
```

Comment:
But now that I think about this some more. Maybe this extension should be in the other direction. Otherwise adding a new protocol requirement would be a breaking change to a type that conforms to DiagnosticConsumer.